packages/launch/src/hooks/use-title.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { useDispatch, useSelect } from '@wordpress/data';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { useEntityProp } from '@wordpress/core-data';
```

Comment:
TODO: add TypeScript types for `useEntityProp` (could be also solved when removing code duplication for  `hooks` folder

Reply:
This should be fixed in https://github.com/Automattic/wp-calypso/pull/46921